addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -182,84 +198,46 @@ const dynamicSnippetOptions = options.Class.extend({
      */
-    _renderCustomXML: async function (uiFragment) {
-        await this._renderDynamicFiltersSelector(uiFragment);
-        await this._renderDynamicFilterTemplatesSelector(uiFragment);
-    },
-    /**
-     * Renders the dynamic filter option selector content into the provided uiFragment.
-     * @param {HTMLElement} uiFragment
-     * @private
-     */
-    _renderDynamicFiltersSelector: async function (uiFragment) {
-        const filtersSelectorEl = uiFragment.querySelector('[data-name="filter_opt"]');
-        return this._renderSelectUserValueWidgetButtons(filtersSelectorEl, this.dynamicFilters);
-    },
-    /**
-     * Renders we-buttons into a SelectUserValueWidget element according to provided data.
-     * @param {HTMLElement} selectUserValueWidgetElement the SelectUserValueWidget buttons
-     *   have to be created into.
-     * @param {Object} data
-     * @private
-     */
-    _renderSelectUserValueWidgetButtons: async function (selectUserValueWidgetElement, data) {
-        for (let id in data) {
-            const button = document.createElement('we-button');
-            button.dataset.selectDataAttribute = id;
-            if (data[id].thumb) {
-                button.dataset.img = data[id].thumb;
-            } else {
-                button.innerText = data[id].name;
-            }
-            selectUserValueWidgetElement.appendChild(button);
-        }
-    },
+    async _getRenderContext() {
+        return {
+            dynamicFilter: this.dynamicFilter || {},
+            dynamicFilterTemplates: this.dynamicFilterTemplates || {},
+        };
+    }
     /**
      * Renders the template option selector content into the provided uiFragment.
      * @param {HTMLElement} uiFragment
      * @private
      */
-    _renderDynamicFilterTemplatesSelector: async function (uiFragment) {
-        const templatesSelectorEl = uiFragment.querySelector('[data-name="template_opt"]');
-        return this._renderSelectUserValueWidgetButtons(templatesSelectorEl, this.dynamicFilterTemplates);
-    },
+    async _renderDynamicFilterTemplatesSelector(uiFragment) {
+        //const templatesSelectorEl = uiFragment.querySelector('[data-name="template_opt"]');
+        //return this._renderSelectUserValueWidgetButtons(templatesSelectorEl, this.dynamicFilterTemplates);
+    }
     /**
      * Sets default options values.
      * Method to be overridden in child components in order to set additional
      * options default values.
      * @private
      */
     async _setOptionsDefaultValues() {
-        // Unactive the editor observer, otherwise, undo of the editor will undo
-        // the attribute being changed. In some case of undo, a race condition
-        // with the public widget that use following property (eg.
-        // numberOfElements or numberOfElementsSmallDevices) might throw an
-        // exception by not finding the attribute on the element.
-        this.options.wysiwyg.odooEditor.observerUnactive();
```

Comment:
Why was this line removed? Shouldn't it still be there?

Reply:
This is only called during `onBuilt` which is already part of the step of dropping the snippet => those operations do not create additional steps. Also, this unactive/active was not applied to overrides of the method.